### PR TITLE
Potted lean update

### PR DESCRIPTION
**Pull request by ghost**
[Changelogs]: # 

:cl: Spankmaster
tweak: Help intent on potted lean now causes you to take a sip from it.
/:cl:

[why]: # Makes the mob more dynamic.

### CODE REVIEW

File: hippiestation/code/modules/mob/living/simple_animal/hostile/retaliate/potted_lean.dm [hunk 1]
```diff
@@ -27,12 +27,19 @@
...
-			
+/mob/living/simple_animal/hostile/retaliate/pottedlean/attack_hand(mob/living/user)
+	if(user.a_intent == INTENT_HELP && user.reagents)
+		visible_message("<span class='notice'>[user] takes a sip from [src]</span>")
+		user.reagents.add_reagent(attack_inject, 10)
+	else
+		..()
+
+
+
```
Comment (JohnGinnane) [inline hippiestation/code/modules/mob/living/simple_animal/hostile/retaliate/potted_lean.dm]: lotta space here

**Comment by JohnGinnane** [comment]
did you test this

**Comment by ghost** [comment]
Yea John for future reference I won’t PR something unless it’s been tested. I’ll fix the spaces when I get home

**Comment by ghost** [comment]
OK, whitespace removed.

**Review by JohnGinnane — APPROVED** [review]

**Comment by JohnGinnane** [comment]
does this need @HippieStation/head input

**Comment by Pyko1** [comment]
why not, +1

**Comment by ghost** [comment]
+1

**Comment by ghost** [comment]
Your Fix/Feature pull request delta is currently below zero (-1). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**